package.json

```diff
@@ -24,6 +24,7 @@
     "trailingComma": "es5"
   },
   "dependencies": {
+    "@newrelic/next": "^0.7.0",
```

Comment:
I think you also want to install the non-next new relic package. This is what DRB is doing: https://github.com/NYPL/sfr-bookfinder-front-end/blob/development/package.json#L26C6-L26C14

Reply:
The documentation was unclear on this. They explicitly say for next apps to install the next agent _instead of_ the newrelic agent. They also say this:

`Currently this package is not bundled with the agent, and must be installed as a standalone. However, the package depends on the agent so you will get all the capabilities of the agent when loading this package.`

On the contrary, their example app shows both installed. 

I'll go ahead and install them both.